cmd/cvc-operator/controller/cvc_controller.go

```diff
@@ -263,6 +275,16 @@ func (c *CVCController) createVolumeOperation(cvc *apis.CStorVolumeClaim) (*apis
 		return nil, err
 	}
 
+	if cvclaim.IsHAVolume(cvc) {
+		var pdbObj *policy.PodDisruptionBudget
+		pdbObj, err = getOrCreatePodDisruptionBudget(cvObj, getCSPC(cvc))
+		if err != nil {
+			return nil, errors.Wrapf(err,
+				"failed to create PDB for volume: %s", cvc.Name)
+		}
+		cvc = addPDBLabelOnCVC(cvc, pdbObj)
```

Comment:
it should be fine even without assigning back.. correct?

Reply:
Updated the function signature